hi.c

```diff
@@ -0,0 +1,7 @@
+#include<stdio.h>
```

Comment:
this is not right

Reply:
this is how you do it noob!
```suggestion
#include<stdio.h> lol
```